nodes/src/nodes/db_neo4j/IGlobal.py

```
@@ -0,0 +1,305 @@
+
+        except Exception as e:
+            warning(f'Neo4J schema reflection failed: {e}')
+
+        return schema
+
+    # ------------------------------------------------------------------
+    # Internal helpers
+    # ------------------------------------------------------------------
+
+    @staticmethod
+    def _build_auth(config: Dict[str, Any]) -> Any:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_build_auth`

(ANN401)